client-app/shared/cart/components/add-to-cart.vue

```diff
@@ -102,7 +102,9 @@ const disabled = computed<boolean>(
     !props.product.availabilityData?.isAvailable ||
     !props.product.availabilityData?.isInStock ||
     !props.product.availabilityData?.isBuyable ||
-    (!props.product.availabilityData?.availableQuantity && !isDigital.value)
+    (!props.product.availabilityData?.isAvailable &&
```

Comment:
!isAvailable && !isDigital is enough here, because if isAvailable = false, availableQantity will be = 0

Reply:
I see. Seems to be a complex logic in a future. This component should be exterminated.